src/libraries/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -310,6 +311,7 @@ public sealed partial class X509Certificate2Enumerator : System.Collections.IEnu
         public void Reset() { }
         bool System.Collections.IEnumerator.MoveNext() { throw null; }
         void System.Collections.IEnumerator.Reset() { }
+        void System.IDisposable.Dispose() { }
```

Comment:
@bartonjs, opinion on this being explicitly implemented vs implicitly?

Reply:
For a sealed type that most people never use directly where it's a no-op merely required by interface inheritance?  Nah.